src/components/Person/Person.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+export function Person(props) {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = props.person;
+
+  const partnerStatus = (sex === 'm')
+    ? 'wife\'s'
+    : 'husband\'s';
```

Comment:
```suggestion
  const partner = (sex === 'm')
    ? 'wife'
    : 'husband';
```
More meaningful naming